xds/src/test/java/com/linecorp/centraldogma/xds/k8s/v1/XdsKubernetesServiceTest.java

```diff
@@ -179,7 +180,9 @@ void invalidProperty() throws IOException {
         watcher = watcher(watcherId, "nginx-service", "invalid-credential-id");
         response = createWatcher(watcher, watcherId);
         assertThat(response.status()).isSameAs(HttpStatus.BAD_REQUEST);
-        assertThat(response.contentUtf8()).contains("failed to find credential 'invalid-credential-id'");
+        assertThatJson(response.contentUtf8())
+                .node("grpc-code").isEqualTo("INVALID_ARGUMENT")
+                .node("message").isEqualTo("failed to find credential 'invalid-credential-id' in @xds/meta");
```

Comment:
This test failed because `Flags.verborseResponses()` is now disabled by default.
Changed `XdsKubernetesService` to set exception messages as gRPC status descriptions.